client/client.go

```diff
@@ -56,6 +56,10 @@ type Client interface {
 	GetLocalTS(ctx context.Context, dcLocation string) (int64, int64, error)
 	// GetLocalTSAsync gets a local timestamp from PD, without block the caller.
 	GetLocalTSAsync(ctx context.Context, dcLocation string) TSFuture
+	// GetLastTS gets the last timestamp given by GetTS.
+	GetLastTS(ctx context.Context) (int64, int64, error)
+	// GetLastLocalTS gets the last timestamp given by GetLastLocalTS.
+	GetLastLocalTS(ctx context.Context, dcLocation string) (int64, int64, error)
```

Comment:
Maybe we can merge these two methods into a single one?

Reply:
Thanks for your review, I agree with you.